backend/src/main/java/ai/verta/modeldb/config/Config.java

```diff
@@ -0,0 +1,55 @@
+package ai.verta.modeldb.config;
+
+import static ai.verta.modeldb.utils.ModelDBUtils.appendOptionalTelepresencePath;
+
+import ai.verta.modeldb.ModelDBConstants;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.constructor.Constructor;
+import java.util.Optional;
+
+public class Config {
+  public static String MISSING_REQUIRED = "required field is missing";
+
+  private static Config config = null;
+
+  public DatabaseConfig database;
+  public TestConfig test;
+
+  // FIXME
```

Comment:
What is still pending to fix?? i can't get it.

Reply:
We need to support the other parts of the config with concrete types instead of the placeholder `Object`.

Reply:
Ok, got it.